fe/fe-core/src/main/java/com/starrocks/sql/optimizer/OutputPropertyDeriver.java

```diff
@@ -228,10 +228,10 @@ private PhysicalPropertySet visitPhysicalJoin(PhysicalJoinOperator node, Express
             } else if ((leftDistributionDesc.isShuffle() || leftDistributionDesc.isShuffleEnforce()) &&
                     (rightDistributionDesc.isShuffle()) || rightDistributionDesc.isShuffleEnforce()) {
                 // shuffle join
-                return computeHashJoinDistributionPropertyInfo(node,
-                        computeShuffleJoinOutputProperty(node.getJoinType(), leftOnPredicateColumns, rightOnPredicateColumns),
-                        leftOnPredicateColumns,
-                        rightOnPredicateColumns, context);
+                PhysicalPropertySet outputProperty = computeShuffleJoinOutputProperty(node.getJoinType(),
```

Comment:
please modify the parameter name in this function `computeShuffleJoinOutputProperty()`